src/utils/update/beneficial_owners_managing_officers_data_fetch.ts

```diff
@@ -58,17 +60,22 @@ export const retrieveBeneficialOwners = async (req: Request, appData: Applicatio
 };
 
 export const retrieveManagingOfficers = async (req: Request, appData: ApplicationData) => {
+  const transactionId = appData.transaction_id;
+  const overseasEntityId = appData.overseas_entity_id;
   const companyOfficers = await getCompanyOfficers(req, appData[EntityNumberKey] as string);
+  const moPrivateData: ManagingOfficersPrivateData = await getManagingOfficerPrivateData(req, transactionId, overseasEntityId) as ManagingOfficersPrivateData;
   if (companyOfficers) {
     for (const officer of (companyOfficers.items || [])) {
       logger.info("Loaded officer " + officer.officerRole);
       if (officer.resignedOn === undefined) {
         if (officer.officerRole === "managing-officer") {
           const managingOfficer = mapToManagingOfficer(officer);
+          managingOfficer.usual_residential_address = mapMoPrivateAddress(moPrivateData, managingOfficer.ch_reference as string);
```

Comment:
Using "as" to assert that there is a ch_reference is bit risky. Maybe don't map if undefined.